backend/inspirehep/records/indexer/tasks.py

```diff
@@ -66,23 +90,8 @@ def index_record(self, uuid, record_version=None, force_delete=None):
     Returns:
         list(dict): Statistics from processing references.
     """
-    try:
-        record = get_record(uuid, record_version)
-    except (NoResultFound, StaleDataError) as e:
-        LOGGER.debug(
-            "Record not yet at version on DB", uuid=str(uuid), version=record_version
-        )
-        backoff = 2 ** (self.request.retries + 1)
-        if self.max_retries < self.request.retries + 1:
-            LOGGER.debug(
-                "Record not yet at version on DB - Too many retries",
-                uuid=str(uuid),
-                version=record_version,
-                attempts=self.max_retries,
-            )
-        raise self.retry(countdown=backoff, exc=e)
-
     LOGGER.debug("Indexing record", uuid=str(uuid), version=record_version)
+    record = get_record(uuid, record_version)
```

Comment:
We can add a ``TODO`` to move this function in ``InspireRecord``

Reply:
:+1: